menu.py

```diff
@@ -22,6 +22,7 @@ def __init__(self):
         blast away the enemy.  Hurry, time is running out!
         '''
         text_writer = LongTextWriter(self.description)
+        text_writer.update()
```

Comment:
Interesting, any reason why the call to update was moved out to the menu class?

Reply:
This was so that I can write unit tests.  I prefer the LongTextWriter class to just do everything in its constructor, but I couldn't think of how to test it then.